lib/leaflet.bing.js

```diff
@@ -1,5 +1,9 @@
 "use strict";
 
+var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
```

Comment:
Dieses Repo hat keine ordentliche Dependency auf `@babel/runtime`. Sollte in unserem Fall aber egal sein, weil wir das sowieso schon haben.